File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03705/p03705_num3_parsed.py
import math

N, A, B = map(int, input().split())

if A == B:
    print(1)
else:
    if N == 1:
        print(0)
    else:
        min_sum = A * (N - 1) + B
        max_sum = A + B * (N - 1)
        print(max_sum - min_sum + 1)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03705/p03705_num3_parsed.cpp
#include <iostream>
using namespace std;

int main() {
    int N, A, B;
    cin >> N >> A >> B;

    if (A == B) {
        cout << 1 << endl;
    } else if (N == 1) {
        cout << 0 << endl;
    } else {
        int min_sum = A * (N - 1) + B;
        int max_sum = A + B * (N - 1);
        cout << max_sum - min_sum + 1 << endl;
    }

    return 0;
}